CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+### Contributing
+
+- Contributions to catch up with latest versions of LSP and Monaco are always welcomed!
+- Contributions to propose a new example should be only accepted if an author is ready to become a maintainer
+and help with upgrading and testing it with latest versions of LSP and Monaco.
+
+### Maintaining
+
+The project is maintained by individuals using for its own purposes and being ready to help as it suites them.
+There is no guruantee on time of response to issues and pull requests.
+
+#### Current Maintainers
+
+- @AlexTugarev - Alex Tugarev
```

Comment:
@AlexTugarev @gatesn @mofux @rcjsuen @asual @zewa666 @johnwiseheart You are all invited as collaborators to this project with `Admin` access. Could you please have a look at this PR, review, test and approve it if you are happy? 🙏 I will merge then, review other PRs and do a release for this time.

Afterward you can do it. I will add you as collaborators to npmjs that you can publish from your npmjs accounts. I need your npmjs accounts first though, please share.

Reply:
👍 my npmjs account is `johnwiseheart`

Reply:
mine is `mofux`

Reply:
npm fails when i try to add you :( I've file a support ticket for npmjs, hope to add you as soon as it is resolved

Reply:
@mofux @johnwiseheart npm support sad that it is not possible, since the package belongs to the organization and only org members can collaborate on it :(

anyway i've configured ci on master to publish next version on commit, so if you merge something you should be able to try it with next